SetupDataPkg/Docs/Overview/Overview.md

```diff
@@ -24,192 +27,153 @@ This document is intended to describe the Setup Variable design on applicable pl
 | Kun Qin   | 09/28/2021| First draft |
 | Oliver Smith-Denny | 7/22/2022 | Add Merged YAML/XML Support |
 | Oliver Smith-Denny | 9/15/2022 | Add Profile Support |
+| Oliver Smith-Denny | 2/21/2023 | Move to XML Spec |
 
 ## Terms
 
 | Term   | Description                     |
 | ------ | ------------------------------- |
 | UEFI | Unified Extensible Firmware Interface |
-| DFCI | Device Firmware Configuration Interface |
 | BDS | Boot Device Selection |
+| FW | Firmware |
+| FMP | Firmware Management Protocol |
 
 ## Reference Documents
 
 | Document                                  | Link                                |
 | ----------------------------------------- | ----------------------------------- |
-| Slim Bootloader Repo | <https://github.com/slimbootloader/slimbootloader> |
-| Configuration YAML Spec | <https://slimbootloader.github.io/specs/config.html#configuration-description-yaml-explained> |
-| Project Mu Document | <https://microsoft.github.io/mu/> |
-| DFCI Documents | <https://microsoft.github.io/mu/dyn/mu_plus/DfciPkg/Docs/Dfci_Feature/> |
-| Configuration Apps Repo | <https://windowspartners.visualstudio.com/MSCoreUEFI/_git/mu_config_apps> |
+| Project Mu Documentation | <https://microsoft.github.io/mu/> |
 
 ## Introduction

```

Comment:
Should this doc call out a comparison to DFCI?

Reply:
Added a brief comparison and pointer to DFCI.